scripts/kill-host-pods.py

```diff
@@ -21,7 +21,7 @@ def post_filter_has_known_containers(pod, containers: list) -> bool:
     Return true if any of the container IDs on the pod match the list of
     containers passed on the second argument.
     """
-    for container in pod["status"]["containerStatuses"] or []:
+    for container in pod["status"].get("containerStatuses") or []:
```

Comment:
nit: Maybe we can change this to:
```suggestion
    for container in pod["status"].get("containerStatuses", []):
```

Reply:
Can `containerStatuses` ever be present-but-null in the YAML, or is it always either present-and-valid or not present at all?

I wrote this as `.get("containerStatuses") or []` because that will handle both cases (missing or present-but-null) and treat them as an empty array.  Using `.get("containerStatuses", [])` will give you an empty array if `containerStatuses` is missing, but will return `None` if it is set-but-null (and in turn raise an error when you try to iterate over a `None` value).